tools/appveyor.psm1

```diff
@@ -225,7 +231,7 @@ function Invoke-AppVeyorInstall
     }
 
     Set-BuildVariable -Name TestPassed -Value False
-    Start-PSBootstrap -Force
+    Start-PSBootstrap
```

Comment:
I noticed that `Start-PSBootStrap` is defined with `ConfirmImpact="High"` and there are two places it asks for confirmation -- installing chocolatey and updating the machine Path environment variable. So will removing `-Force` cause any problems?